packages/fluentui/docs/src/examples/components/Popup/Types/PopupExampleInline.tsx

```diff
@@ -1,11 +0,0 @@
-import * as React from 'react';
-import { Button, Popup } from '@fluentui/react-northstar';
-import { MoreIcon } from '@fluentui/react-icons-northstar';
-
-const PopupExampleInline = () => (
-  <Popup content="This popup is rendered next to the trigger." inline>
-    <Button icon={<MoreIcon />} title="Show popup" />
-  </Popup>
-);
-
-export default PopupExampleInline;
```

Comment:
Just wondering, why these examples are removed?

Reply:
Because for what I understood non of these files were being used, all the example still there since there was the same files but with shorthand in the end `PopupExampleInline .shorthand` and exactly same code... 

If I am wrong just let me know and I will revert.

Reply:
Reverted. Thanks @pompomon for the explanations :D